### PR TITLE
feat(desktop-updater): Server

### PR DESCRIPTION
**Pull request by pwnsdx**

### CODE REVIEW

File: packages/desktop-updater-server/src/main/Server.ts [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import * as sodium from 'sodium-native';
```
Comment (ffflorian) [inline packages/desktop-updater-server/src/main/Server.ts]: Why not use `libsodium-wrappers`?

File: packages/desktop-updater-server/src/main/Utils.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
```
Comment (ffflorian) [inline packages/desktop-updater-server/src/main/Utils.ts]:
```suggestion
 * Copyright (C) 2019 Wire Swiss GmbH
```

File: packages/desktop-updater-server/src/main/Server.ts (hunk 1 above)
Comment (pwnsdx) [reply packages/desktop-updater-server/src/main/Server.ts]: I believe `libsodium-wrappers` was not existing when the project was created. Changing the sodium library over this project would be a huge work so I think we should go for `sodium-native` for now.

Comment (ffflorian) [reply packages/desktop-updater-server/src/main/Server.ts]: Or probably it didn't have WASM back then..

Comment (ffflorian) [reply packages/desktop-updater-server/src/main/Server.ts]: > Changing the sodium library over this project would be a huge work

You are using only one function, where is the huge work?

Comment (pwnsdx) [reply packages/desktop-updater-server/src/main/Server.ts]: > > Changing the sodium library over this project would be a huge work
> 
> You are using only one function, where is the huge work?

Changing it here will create the need to change it everywhere such as here: https://github.com/wireapp/wire-desktop-packages/pull/314 and https://github.com/wearezeta/wire-desktop-updater-toolkit

**Pull request closed without merging**